pages/dkp/kommander/2.1/release-notes/2.1.2/index.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.2 Release Notes
+title: Kommander 2.1.2 Release Notes
+menuWeight: 30
+excerpt: View release-specific information for Kommander 2.1.2
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.2 was released on May 11, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## Fixes and Improvements
+
+### Flux proxy configuration results in failure to install applications (COPS-7236)
+
+Incorrect Flux proxy configuration was resulting in Kommander applications not pulling and installing.
+
+### Broken FIPS images (COPS-7225)
+
+FIPS images for DKP were overwritten with blank docker images, making deployment fail.
+
+### kube-oidc-proxy error: certificate signed by unknown authority (COPS-7217)
+
+When using a custom domain and TLS certificate issued by Let's Encrypt, the `kube-oidc-proxy` helm chart in the attached cluster did not complete installation and the associated pod has an error.
+
+### Certificate objects updated but not reloading in Kommander pods (COPS-7212)
+
+When looking at the certificate objects from a bundle, none are expired but the pods in the cluster do not reload with current/new certificates, and also display that the certificates were expired.
+
+### Konvoy-image-builder 1.7.0 fails in certain situatios (COPS-7207)
+
+This issue is resolved.
+
+### KIB fails to pull images in airgapped using ct (COPS-7198)
+
+This lead to a problem in air-gapped environments where the images failed to pull in KIB, when using registry mirrors and/or credentials.
+
+### Dashboards redirecting to wrong URL (COPS-7197)
+
+The `traefik-forward-auth-kommander` helm value was overriding ConfigMap with an incorrect authHost value.
+
+### Kommander fails after a clean reinstall (COPS-7193)
+
+There was an unintended name collision between two separate secrets.
```

Comment:
Confirmed with Martin from EU team. Should we delete it completely, or say it is fixed in 2.2 and later versions @cneth ? I think it would be helpful to know an upgrade fixes this. But this RN already feel pretty full with information...

Reply:
We should just take this out. We will document it in the 2.2.1 release notes.